proxy/proxy_handlers.go

```diff
@@ -0,0 +1,112 @@
+package proxy
+
+import (
+	"bufio"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/websocket"
+	"github.com/taskcluster/webhooktunnel/util"
+	"github.com/taskcluster/webhooktunnel/wsmux"
+)
+
+func (p *proxy) handler(w http.ResponseWriter, r *http.Request) {
+	// register a worker
+	p.logger.Printf("new request: %s, Method: %s", r.URL.Path, r.Method)
+	if strings.HasPrefix(r.URL.Path, "/register") {
```

Comment:
maybe use gorilla mux instead... or match with regular expressions... so you are more strict...